real_estate_module/models/estate_property.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True)              # Property name
+    create_uid=fields.Char(required=True)
+    create_date=fields.Char(required=True)
```

Comment:
```suggestion
    create_uid = fields.Char(required=True)
    create_date = fields.Char(required=True)
```
styling remark, and please go through this [documentation](https://www.odoo.com/documentation/18.0/developer/tutorials/backend.html?highlight=module%20name#reserved-fields) too